java/ql/lib/ext/java.nio.file.model.yml

```diff
@@ -18,6 +18,7 @@ extensions:
       - ["java.nio.file", "Files", False, "delete", "(Path)", "", "Argument[0]", "create-file", "ai-manual"] # should be delete-file
       - ["java.nio.file", "Files", False, "deleteIfExists", "(Path)", "", "Argument[0]", "create-file", "ai-manual"] # should be delete-file
       - ["java.nio.file", "Files", False, "deleteIfExists", "(Path)", "", "Argument[0]", "create-file", "ai-manual"] # should be delete-file
+      - ["java.nio.file", "Files", False, "getFileStore", "(Path)", "", "Argument[0]", "sinkModel/read-file", "ai-manual"] # the FileStore class is unlikely to be used for later sanitization
```

Comment:
This `sinkModel/` prefix doesn't look right.

Reply:
our bad — fixed! thank you!